### PR TITLE
[Docs] Japanese translation of docs/flashing_bootloadhid.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add a Japanese translation of docs/flashing_bootloadhid.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/flashing_bootloadhid.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+1. 以下のいずれかの方法を使ってブートローダに入ります:
+   * `RESET` キーコードをタップします (全てのデバイスでは動作しないかもしれません)
+   * キーボードを接続しながらソルトキーを押し続けます (通常はキーボードの readme に書かれています)
+2. OS がデバイスを検知するのを待ちます。
+3. .hex ファイルを書き込みます
+4. デバイスをアプリケーションモードにリセットします(自動的に実行されるかもしれません)
+
+## bootloadHID の書き込みターゲット
+
+?> [こちら](ja/newbs_getting_started.md)で詳しく説明されている QMK インストールスクリプトを使って、必要な bootloadHID ツールを自動的にインストールしていなければなりません。
```
Comment (shelaf) [inline docs/ja/flashing_bootloadhid.md:21]: should beを～されるはずだ、と訳すと、全体はこんな感じになりそうです
```suggestion
?> [こちら](ja/newbs_getting_started.md)で詳しく説明されている QMK インストールスクリプトを使うと、必要な bootloadHID ツールが自動的にインストールされます。
```

File: docs/ja/flashing_bootloadhid.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+2. キーボードをリセットします。
+3. 設定された VendorID が `16c0` で、ProductID が `05df` であることを確認します
+4. `Find Device` ボタンを押し、キーボードが見つかることを確認します。
+5. `Open .hex File` ボタンを押し、作成した `.hex` ファイルを見つけます。
+6. `Flash Device` ボタンを押し、処理が完了するまで待ちます。
+
+## コマンドライン書き込み
+
+1. キーボードをリセットします。
+2. `bootloadHID -r` に続けて `.hex` ファイルへのパスを入力し、ボードに書き込みます。
```
Comment (shelaf) [inline docs/ja/flashing_bootloadhid.md:40]:
```suggestion
2. `bootloadHID -r` に続けて `.hex` ファイルへのパスを入力し、キーボードに書き込みます。
```

File: docs/ja/flashing_bootloadhid.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+### Windows 手動インストール
+MSYS2の場合:
+1. https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz から BootloadHID ファームウェアをダウンロードします。
```
Comment (shelaf) [inline docs/ja/flashing_bootloadhid.md:44]:
```suggestion
1. https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz から BootloadHID ファームウェアパッケージをダウンロードします。
```

File: docs/ja/flashing_bootloadhid.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+2. 互換性のあるツール、例えば 7-Zip を使って内容を抽出します。
+3. 解凍された書庫から MSYS2インストール、一般的に `C:\msys64\usr\bin` に `commandline/bootloadHID.exe` をコピーして、MSYS パスに追加します。
```
Comment (shelaf) [inline docs/ja/flashing_bootloadhid.md:46]:
```suggestion
3. 解凍された書庫から MSYS2 インストール先、通常 `C:\msys64\usr\bin` に `commandline/bootloadHID.exe` をコピーして、MSYS パスに追加します。
```

File: docs/ja/flashing_bootloadhid.md (hunk 1 above)
Comment (umi-umi) [reply docs/ja/flashing_bootloadhid.md]: 訳例ありがとうございます。

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**